### DISCUSSION TITLE
AppleSignInAuthenticator Scopes.

### DISCUSSION THREAD

**Discussion by Quetzalcoatl2435** [dotnet/maui#14974] [Q&A] [answered]
Hi, I'm using `AppleSignInAuthenticator.AuthenticateAsync` with `IncludeEmailScope` and `IncludeFullNameScope` enabled in my iOS app, but `name` and `email` in `result.Properties` always return null.

Is there any other way to get the user's name and e-mail address from Apple sign in?

**Comment by FM1973** [comment]
As far as I can remember, email and fullname are only provided the first time you use the appleid for logging in or registering. You should then store the information. If you need it again (from apple), then you have to remove the apple id from the app.

**Reply by Quetzalcoatl2435** [reply]
I have deleted the app from the iPhone simulator apple id settings, but still getting null on registering. I might have to play around with a real device and see how that goes.

**Comment by Quetzalcoatl2435** [comment, marked as answer]
Couldn't get WebAuthenticator working as expected, so used https://www.xamboy.com/2020/01/13/sign-in-with-apple-in-xamarin-forms/ instead.
